resources/user-agents/browsers/samsung/samsung-android-browser-6-2-on.json

```diff
@@ -1,6 +1,6 @@
 {
   "division": "Samsung Browser #MAJORVER#.#MINORVER#",
-  "versions": ["13.2", 13, "12.1", 12, "11.2", "11.1", 11, "10.2", "10.1", 10, "9.4", "9.3", "9.2", 9, "8.3", "8.2", 8, "7.4", "7.2", 7, "6.4", "6.3", "6.2"],
+  "versions": [16, 15, "14.2", 14, "13.2", 13, "12.1", 12, "11.2", "11.1", 11, "10.2", "10.1", 10, "9.4", "9.3", "9.2", 9, "8.3", "8.2", 8, "7.4", "7.2", 7, "6.4", "6.3", "6.2"],
```

Comment:
is 14.1 not missing here? Or was it never released as version for samsung browser?

Reply:
I have not found any 14.1 version in our UA logs, but I can add them just in case (but for example, there is not a 13.1 either)